racktables/get_unused_machines.py

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env python3
+from racktables import Racktables, RacktablesObject
+from getpass import getpass
+import os
+
+rt_url = os.environ["RT_URL"] if "RT_URL" in os.environ.keys() else "https://racktables.suse.de"
```

Comment:
If is possible I'd suggest to change these three lines like this:
```suggestion
rt_url = os.environ.get("RT_URL", "https://racktables.suse.de")
```

Reply:
I'd love to have used this pattern for the other two values as well…